documentation/FAQ.md

```diff
@@ -185,8 +185,17 @@ def toy_example(a, b):
    ...
 ```
 
+## Why didn't my code recompile when I changed it?
+
+If you went ahead and enabled dynamic shapes via `env TORCHDYNAMO_DYNAMIC_SHAPES=0 python model.py`
+then your code won't recompile on shape changes. We've added support for dynamic shapes which avoids recompilations in the case when shapes vary by less than a factor of 2. This is especially useful in scenarios like varying image sizes in CV or variable sequence length in NLP. In inference scenarios it's often not possible to know what a batch size will be beforehand because you take what you can get from different client apps.
```

Comment:
Wait, really?! News to me lol

Reply:
@soumith I've heard you say that factor of 2 compilations are avoided and that partial graph changes would only cause partial graph recompilations. Did I mishear?

Reply:
triton doesn't recompile unless it's the next power-of-two shape (or if it's a different memory alignment, but that's rare)